scripts/cleanup_datasets/cleanup_datasets.py

```diff
@@ -106,7 +106,7 @@ def main():
     now = strftime("%Y-%m-%d %H:%M:%S")
 
     log.info("##########################################")
-    log.info("\n# %s - Handling stuff older than %i days" % (now, args.days))
+    log.info("\n# %s - Handling stuff older than %d days" % (now, args.days))
```

Comment:
Actually the preferred syntax is
```suggestion
    log.info("\n# %s - Handling stuff older than %d days", now, args.days)
```

See https://docs.python.org/3/howto/logging.html#logging-variable-data

Reply:
Indeed, it is. Of course all of them should be changed then.